src/CapStatus.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import styles from "./CapStatus.module.css"
+import rocketActive from "./assets/rocketActive.svg"
+import rocketUnknown from "./assets/rocketUnknown.svg"
+import rocketDeactivate from "./assets/rocketDeactivate.svg"
```

Comment:
```suggestion
import React from "react"

import styles from "./CapStatus.module.css"

import rocketActive from "./assets/rocketActive.svg"
import rocketUnknown from "./assets/rocketUnknown.svg"
import rocketDeactivate from "./assets/rocketDeactivate.svg"
```
Tal vez un par de enters